### ISSUE TITLE
Add proprietary webkit pseudo-classes to list of known ones in selector-pseudo-class-no-unknown

### ISSUE THREAD

**Issue by AoDev** [stylelint/stylelint#2257]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

This selector will fail `::-webkit-scrollbar-thumb:window-inactive`

> Which rule, if any, is this issue related to?

selector-pseudo-class-no-unknown. (it does not like the `:window-inactive`)

> What CSS is needed to reproduce this issue?

e.g.

```css
::-webkit-scrollbar-thumb:window-inactive {
  background: red;
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "selector-pseudo-class-no-unknown": true
  }
}
```

> Which version of stylelint are you using?

7.7.1

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Both CLI and Sublime Text stylelint contrib plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

My files are written in less but this rule in particular is not using anything special. It is not nested.

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
[{"line":24,"column":26,"rule":"selector-pseudo-class-no-unknown","severity":"error","text":"Unexpected unknown pseudo-class selector \":window-inactive\" (selector-pseudo-class-no-unknown)"}]
```

I haven't checked stylelint code but maybe it's just forgotten.  
Note: it does not fail for mozilla specific.

```
:-moz-window-inactive {
  background: red;
}
```

**Comment by jeddy3** [comment]
@AoDev Thanks for the report and for using the template.

That's a peculiar pseudo class: non-standard but _unprefixed_. I think we ought to account for it though.

Would you like [to contribute](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md) a new set of the 11 pseudo-classes from [this webkit blog post](https://webkit.org/blog/363/styling-scrollbars/)? 

1. You can add the new set [here](https://github.com/stylelint/stylelint/blob/master/lib/reference/keywordSets.js#L254) and call it something like `keywordSets.webkitProprietaryPseudoClasses`. 
2. Remember to unite it [here](https://github.com/stylelint/stylelint/blob/master/lib/reference/keywordSets.js#L258).
3. Finally, add a test for [here](https://github.com/stylelint/stylelint/blob/master/lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js#L48) for `::-webkit-scrollbar-thumb:window-inactive {}`.
4. Submit a PR.

**Comment by AoDev** [comment]
Hi, currently I don't have much time, I could check it later. I've seen someone is assigned so, do you still need me?

**Comment by evilebottnawi** [comment]
@AoDev already done in https://github.com/stylelint/stylelint/pull/2264 PR :smile:

**Comment by evilebottnawi** [comment]
Done in https://github.com/stylelint/stylelint/pull/2264
